rules/config

```diff
@@ -76,7 +76,12 @@ SONIC_ROUTING_STACK = frr
 
 # ENABLE_SYNCD_RPC - build docker-syncd with rpc packages for testing purposes.
 # Uncomment to enable:
-# ENABLE_SYNCD_RPC = y
+ENABLE_SYNCD_RPC = y
```

Comment:
Do we need to keep this enabled by default? Same below.

Reply:
Fixed